apps/bluetooth/locales/bluetooth.en-US.properties

```diff
@@ -41,7 +41,7 @@ search-for-device   = Searching for devices…
 search-device       = Search for devices
 rename-device       = Rename my device
 unnamed-device      = Unknown device
-change-phone-name   = Change phone name
+change-phone-name   = Rename My Device
```

Comment:
It's not allowed to change strings without changing l10nid key. Even more, almost the same text already defined 2 lines above (see line 42: `` rename-device       = Rename my device``). According to David's comment, Mozilla UX team prefer not to capitalize first letters in all words in sentence, so while strings are not 100% defined & approved by Mozilla UX it's better to stick to Mozilla's practices.
This comment also applies to changes in lines 56-58 in this file.